src/Illuminate/Database/Concerns/ManagesTransactions.php

```diff
@@ -331,4 +348,21 @@ public function afterCommit($callback)
 
         throw new RuntimeException('Transactions Manager has not been set.');
     }
+
+    /**
+     * Set isolation level for current transaction.
+     *
+     * @param  string  $isolationLevel
+     * @return void
+     */
+    protected function prepareIsolationLevel($isolationLevel)
+    {
+        if ($this->getDriverName() === 'sqlite') {
```

Comment:
Instead of bailing only in case of sqlite, wouldn't it be smarter to implement a switch case, with fallback also returning in case of an unknown database driver being used?

I may be wrong, but some other database driver could pose a problem that doesn't support the feature might break functionality(although it'd entirely be the implementer's fault). I'm well aware Laravel Officially only support MySQL, PG, MSSQL and SQLite. Within official support, though, the argument is moot.

What are your thoughts on it?

Reply:
I guess with the new commit, it got fixed.